src/level/LevelDirector.js

```diff
@@ -111,7 +111,7 @@
                         );
                     } else {
                         obj = me.pool.pull(
-                            "me.Entity",
+                            me.pool.exists(settings.className) ? settings.className : "me.Entity",
```

Comment:
Isn't the collision name only for the layer-group? I've got several objects within the collision group with the name "solid" that work just fine